roles/generate-jenkins/templates/Jenkinsfile.j2

```
@@ -391,7 +391,7 @@ pipeline {
 {% if github_project_name != "docker-jenkins-builder" %}
               docker pull linuxserver/jenkins-builder:latest
 {% endif %}
-              docker run --rm -e CONTAINER_NAME=${CONTAINER_NAME} -e GITHUB_BRANCH={{ ls_branch }} -v ${TEMPDIR}:/ansible/jenkins {% if github_project_name != "docker-jenkins-builder" %}linuxserver/jenkins-builder:latest{% else %}jenkinslocal:${COMMIT_SHA}-${BUILD_NUMBER}{% endif %} 
```

Check for these spaces they are needed for templating

---

Would this work?
```suggestion
docker run --rm -e CONTAINER_NAME=${CONTAINER_NAME} -e GITHUB_BRANCH={{ ls_branch }} -v ${TEMPDIR}:/ansible/jenkins {% if github_project_name != "docker-jenkins-builder" %}linuxserver/jenkins-builder:latest{% else %}jenkinslocal:${COMMIT_SHA}-${BUILD_NUMBER}{% endif %} {{ ' ' }}
```
Ref https://jinja.palletsprojects.com/en/2.11.x/templates/#escaping

---

not sure you can build and template locally

---

it is about the line break though not the space

---

I'll test add report.

---

Tested and confirmed.